core/src/test/java/nl/nn/adapterframework/pipes/IncreaseIntegerPipeTest.java

```diff
@@ -45,4 +46,32 @@ public void testCannotIncreaseBy2AsNoSessionKey() throws Exception {
         fail("this is expected to fail");
     }
 
+    @Test
+    public void testIncrementParameter() throws Exception {
+    	String numberSession = "number";
+		session.put(numberSession, "4");
+		Parameter inc = new Parameter();
+		inc.setName("increment");
+		inc.setValue("5");
+		pipe.addParameter(inc);
+		pipe.setSessionKey(numberSession);
+		pipe.configure();
+		doPipe(pipe, "message", session);
+		assertEquals("9", session.get(numberSession));
+    }
+
+    @Test
+    public void testNullIncrementParameter() throws Exception {
+    	String numberSession = "number";
+		session.put(numberSession, "4");
+		Parameter inc = new Parameter();
+		inc.setName("increment");
+		inc.setValue(null);
+		pipe.addParameter(inc);
+		pipe.setSessionKey(numberSession);
+		pipe.configure();
+		doPipe(pipe, null, session);
+		assertEquals("5", session.get(numberSession));
+    }
```

Comment:
For the sake of completeness, could you also add a test with an empty String?
`testEmptyIncrementParameter()`